EDU-2517-become-isv (#386)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/referencia/marketplace/isv-signup.mdx

Para publicar seu produto no Marketplace da Azion, você deve se tornar um Independent Software Vendor (ISV), ou fornecedor independente de software, seguindo o **processo de cadastro de ISV**, conforme explicado nas próximas seções.

<Button href="/pt-br/documentacao/produtos/marketplace/" text="Saiba mais sobre o Marketplace da Azion" variant="secondary">

</Button>
<br/>
<Button href="/pt-br/documentacao/produtos/marketplace/marketplace-seller-guide/" text="Consulte o Marketplace Seller Guide" variant="secondary">

</Button>

---

## Pré-requisitos

- Ter uma [conta na Azion](/pt-br/documentacao/produtos/contas/criar-uma-conta/).
- Concluir o [processo de cadastro e aprovação](#se-cadastrar-como-um-isv) para se tornar um fornecedor.
- Para ser elegível para anunciar seu produto com uma cobrança associada, você deve poder receber pagamentos da Azion e atender aos seguintes requisitos:
   - Sua organização precisa ter *status legal nos Estados Unidos*.
   - Sua organização deve ter uma *conta bancária nos Estados Unidos*.

> :::caution[Warning]

Reviewer: ```suggestion
:::caution[atenção]
```

O​⁠ ​Marketplace da​ Azion não aceita empresas terceirizadas para revenda de produtos.
:::

---

## Se cadastrar como um ISV

> Para iniciar seu cadastro, você deve preencher o [formulário de inscrição de ISV](https://docs.google.com/forms/d/e/1FAIpQLScrOc_FyjVC11VFbVgDjuh6THh2jJs0-I8vduW0nZSCDGEPIg/viewform). Este formulário exige que você forneça informações básicas sobre sua organização e aceite o [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/), que inclui os termos e condiciones.

Reviewer 2: ```suggestion
Para iniciar seu cadastro, você deve preencher o [formulário de inscrição de ISV](https://docs.google.com/forms/d/e/1FAIpQLScrOc_FyjVC11VFbVgDjuh6THh2jJs0-I8vduW0nZSCDGEPIg/viewform). Este formulário exige que você forneça informações básicas sobre sua organização e aceite o [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/), que inclui os termos e condições.
```

Campos marcados com asterisco são obrigatórios.

Onde:

> | **Campo** | **Descrição** |

Author: ```suggestion
| Campo | Descrição |
```

| --- | --- |

> | Email* | Um e-mail para entrar em contato com você. Não será visível para os clientes quando o seu produto for publicado no Marketplace |

Reviewer: ```suggestion
| Email* | Um email para entrar em contato com você. Não será visível para os clientes quando o seu produto for publicado no Marketplace |
```

| Contact Name | Nome do responsável pelo contrato com a Azion. Não será visível para os clientes quando o seu produto for publicado no Marketplace |
| Company Name or Developer Name* | Nome do criador do seu produto. Será exibido quando o seu produto for publicado no Marketplace |
| Website | Seu website. Será exibido quando o seu produto for publicado no Marketplace |

> | Company Logo | Um URL contendo o arquivo do seu logotipo que será exibido quando seu produto for publicado no marketplace. Deve ser um arquivo `JPG` ou `PNG` com largura máxima de `120px` |

Reviewer: ```suggestion
| Company Logo | Uma URL contendo o arquivo do seu logotipo que será exibido quando seu produto for publicado no Marketplace. Deve ser um arquivo `JPG` ou `PNG` com largura máxima de `120px` |
```

| Phone | Um número de telefone para entrar em contato com você. Não será visível para os clientes |
| Azion Marketplace Agreement* | Os termos e condições [do contrato com o Marketplace](/en/documentation/agreements/marketplace/marketplace-agreement/). Você deve ler o documento e marcar a caixa para concordar com os termos |

Quando tiver preenchido o formulário, clique no botão **Submit**.

A equipe da Azion receberá seu formulário de inscrição.

:::tip
Ative a opção **Send me a copy of my responses** para receber uma cópia da sua inscrição no email fornecido.
:::

---

## Registrar seu produto

> Depois de concluir seu cadastro, é hora de registrar seu produto. Para fazer isso, você deve preencher o [formulário de registro do produto](https://docs.google.com/forms/d/e/1FAIpQLSeoTbF2Qifr7qqHjoaxv9ujFB1Ak4xg6wqV0WfZfpBrpkQ12A/viewform). Este formulário exige que você forneça detalhes importantes sobre seu produto e informações que serão necessárias para publicá-lo no Marketplace da Azion.

Reviewer: ```suggestion
Depois de concluir seu cadastro, você deve registrar seu produto. Para isso, você deve preencher o [formulário de registro do produto](https://docs.google.com/forms/d/e/1FAIpQLSeoTbF2Qifr7qqHjoaxv9ujFB1Ak4xg6wqV0WfZfpBrpkQ12A/viewform). Este formulário exige que você forneça detalhes importantes sobre seu produto e informações que serão necessárias para publicá-lo no Marketplace da Azion.
```

Campos marcados com asterisco são obrigatórios.

O formulário inclui três seções:

### ISV's information

Esta seção requer o nome do ISV e um email válido para entrar em contato com você como responsável pelo produto.

Onde:

> | **Campo** | **Descrição** |

Author: ```suggestion
| Campo | Descrição |
```

| --- | --- |
| Email* | Um email para entrar em contato com você como responsável pelo produto |
| ISV Name* | Seu nome como ISV e criador do produto. Recomenda-se usar o mesmo no campo **Company Name or Developer Name*** no [formulário de inscrição do ISV](#se-cadastrar-como-um-isv) |

### Preparing your product

Esta seção requer informações básicas para serem exibidas na página do seu produto no Marketplace da Azion.

Onde:

> | **Campo** | **Descrição** |

Author: ```suggestion
| Campo | Descrição |
```

| --- | --- |
| Product Name* | Nome do seu produto que será exibido no card e na página de informações do produto no Marketplace |

> | Product Version* | A versão do seu produto que estará disponível no mercado. Exemplo `1.1.0` |

Reviewer: ```suggestion
| Product Version* | A versão do seu produto que estará disponível no mercado. Exemplo: `1.1.0` |
```

| Category* | A categoria do Marketplace da Azion que melhor se adapta ao seu produto. Selecione *pelo menos uma opção* |

> | Pricing model* | O modelo de preços que melhor se adapta ao seu produto. Você pode escolher entre os modelos de licença `Free`(Grátis) or `Bring your own license` (Traga sua licença). Mais informações no [Marketplace Seller Guide](https://www.azion.com/en/documentation/products/marketplace/marketplace-seller-guide/) |

Reviewer 2: ```suggestion
| Pricing model* | O modelo de preços que melhor se adapta ao seu produto. Você pode escolher entre os modelos de licença `Free`(Grátis) or `Bring your own license` (Traga sua licença). Mais informações no [Marketplace Seller Guide](/pt-br/documentacao/produtos/marketplace/marketplace-seller-guide/) |
```

### Describing your product

Esta seção inclui as informações adicionais que serão exibidas na página do seu produto no Marketplace da Azion, facilitando o entendimento dos clientes.

Onde:

> | **Campo** | **Descrição** |

Author: ```suggestion
| Campo | Descrição |
```

| --- | --- |

> | Headline* | Um resumo do seu produto, com até `80 caracteres` |

Reviewer 2: ```suggestion
| Headline* | Um resumo do seu produto de até `80 caracteres` |
```

| Overview* | Uma descrição do seu produto, com até `4.000 caracteres`. Você pode explicar a finalidade do seu produto e quais casos de uso ele visa resolver, bem como qualquer link para conteúdo adicional, como documentação, vídeos e sites, entre outros |
| Usage information* | Uma explicação de como usar seu produto, incluindo pré-requisitos e primeiros passos. Você pode incluir conteúdo adicional, como documentação, webinars, instruções, contratos de usuário e Termos de Serviço, entre outros |
| Support* | Uma descrição do suporte fornecido a este produto específico, incluindo seus canais de atendimento |

Quando tiver preenchido o formulário, clique no botão **Submit**.

A equipe da Azion receberá o registro do seu produto.

---

## Receber a aprovação

A última etapa é aguardar pela aprovação. A equipe da Azion analisará sua inscrição como ISV e seu registro de produto para determinar se tudo está em conformidade e se seu produto é adequado para o Marketplace da Azion.

> A equipe entrará em contato com você por meio do e-mail registrado para confirmar a aprovação e explicar as etapas adicionais para concluir a publicação do seu produto no Marketplace da Azion.

Reviewer: ```suggestion
A equipe entrará em contato com você por meio do email registrado para confirmar a aprovação e explicar as etapas adicionais para concluir a publicação do seu produto no Marketplace da Azion.
```

:::tip

> Para obter mais informações sobre requisitos e condições para publicar seus produtos no Marketplace, consulte o [Marketplace Seller Guide](https://www.azion.com/en/documentation/products/marketplace/marketplace-seller-guide/).

Reviewer 2: ```suggestion
Para obter mais informações sobre requisitos e condições para publicar seus produtos no Marketplace, consulte o [Marketplace Seller Guide](/pt-br/documentacao/produtos/marketplace/marketplace-seller-guide/).
```

---

> 

Author: ```suggestion
```
